src/udev/udev-builtin-net_id.c

```diff
@@ -161,6 +167,67 @@ static struct udev_device *skip_virtio(struct udev_device *dev) {
         return parent;
 }
 
+static int get_virtfn_info(struct udev_device *dev, struct netnames *names, struct virtfn_info *vf_info) {
+        struct udev *udev;
+        const char *physfn_link_file;
+        _cleanup_free_ char *physfn_pci_syspath = NULL;
+        _cleanup_free_ char *virtfn_pci_syspath = NULL;
+        struct dirent *dent;
+        _cleanup_closedir_ DIR *dir = NULL;
+        struct virtfn_info vf_info_local = {};
+        int rc;
```

Comment:
nitpick: pretty much across all our codebase we always call the variable we assign syscall return values and errors to just "r", hence would be cool if you could follow this here, and rename rc → r, but doesn't matter